src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Combination/paginated_list_header.html.twig

```diff
@@ -45,9 +45,14 @@
       >
         {{ 'Edit %combinations_number% combinations'|trans({'%s': ''}, 'Admin.Catalog.Feature') }}
       </button>
+      {% if isMultistoreActive %}
+        {% set deleteLabel = 'Delete %combinations_number% combinations for shop "%shop%"'|trans({'%shop%': contextShopName}, 'Admin.Catalog.Feature') %}
+      {% else %}
+        {% set deleteLabel = 'Delete %combinations_number% combinations'|trans({}, 'Admin.Catalog.Feature') %}
```

Comment:
Similar to the comment above, maybe the confirmation message should be adapted as well `Delete 3 combinations for shop "Default"?`

Reply:
Oh of course I meant only for the multishop case, but you adapted the label of the button but not the message in the modal, that's the one I'm talking about and only in multi shop mode

Reply:
I think we should have a clear display if it impacts all the shops or not.
- For my current shop
Are you sure you want to delete selected combinations for shop “Default”?
- For all shops
Are you sure you want to delete selected combinations for all shops?

Reply:
Ok so to summarize @zuk3975 for single shop delete button you'd need to adapt the `data-modal-message` content just like you did for the `data-btn-label` And for the delete for all shops since it's only displayed in multishop mode anyway you can change its `data-modal-message` attribute directly in the template